packages/gatsby-cli/src/reporter/reporters/ink/components/error.js

```diff
@@ -3,6 +3,40 @@ import path from "path"
 import { Color, Box } from "ink"
 import { get } from "lodash"
 
+const colors = {
```

Comment:
I'll remove this -- I was testing with a structured "error" of warning.

Reply:
We need `theme-ui` theme for CLI!

Reply:
lol! Design tokens and theme-ui for our CI, I like it!